src/test/run-pass/sync-send-iterators-in-libcollections.rs

```diff
@@ -50,11 +50,11 @@ fn main() {
     all_sync_send!(BinaryHeap::<usize>::new(), iter, drain, into_iter);
 
     all_sync_send!(BTreeMap::<usize, usize>::new(), iter, iter_mut, into_iter, keys, values);
-    is_sync_send!(BTreeMap::<usize, usize>::new(), range((Included(&0), Included(&9))));
-    is_sync_send!(BTreeMap::<usize, usize>::new(), range_mut((Included(&0), Included(&9))));
+    is_sync_send!(BTreeMap::<usize, usize>::new(), range((Included(0), Included(9))));
+    is_sync_send!(BTreeMap::<usize, usize>::new(), range_mut((Included(0), Included(9))));
```

Comment:
Was this change required to get this compiling? If so this may be a breaking change?